repos/system_upgrade/common/actors/opensshpermitrootlogincheck/actor.py

```diff
@@ -20,13 +21,21 @@ class OpenSshPermitRootLoginCheck(Actor):
     OpenSSH no longer allows root logins with password.
 
     Check the values of PermitRootLogin in OpenSSH server configuration file
-    and warn about potential issues after update.
+    and warn about potential issues after update to RHEL8.
+
+    The RHEL8 still provided default configuration that allowed root logins,
+    which can lead to possible unwanted changes during the upgrade
     """
     name = 'openssh_permit_root_login'
     consumes = (OpenSshConfig, )
     produces = (Report, )
     tags = (ChecksPhaseTag, IPUWorkflowTag, )
 
+    common_resources = [
```

Comment:
whether it works, I dont know. Tried with copr repo on rhel8, but I am lost at the following step, which fails for me for no obvious reason when I follow the docs:

https://leapp.readthedocs.io/en/latest/el7toel8/actor-rhel7-to-rhel8.html#executing-a-single-actor-that-uses-the-workflow-config

```
# snactor run --save-output IPUWorkflowConfig
Process Process-101:
Traceback (most recent call last):
  File "/usr/lib64/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib64/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.6/site-packages/leapp/repository/actor_definition.py", line 72, in _do_run
    actor_instance.run(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/leapp/actors/__init__.py", line 289, in run
    self.process(*args)
  File "/usr/share/leapp-repository/repositories/system_upgrade/common/actors/ipuworkflowconfig/actor.py", line 19, in process
    ipuworkflowconfig.produce_ipu_config(self)
  File "/usr/share/leapp-repository/repositories/system_upgrade/common/actors/ipuworkflowconfig/libraries/ipuworkflowconfig.py", line 77, in produce_ipu_config
    target=target_version
  File "/usr/lib/python3.6/site-packages/leapp/models/__init__.py", line 90, in __init__
    getattr(defined_fields[field], init_method)(kwargs, field, self)
  File "/usr/lib/python3.6/site-packages/leapp/models/fields/__init__.py", line 110, in from_initialization
    self._validate_model_value(value=source_value, name=name)
  File "/usr/lib/python3.6/site-packages/leapp/models/fields/__init__.py", line 179, in _validate_model_value
    super(BuiltinField, self)._validate_model_value(value, name)
  File "/usr/lib/python3.6/site-packages/leapp/models/fields/__init__.py", line 60, in _validate_model_value
    raise ModelViolationError('The value of "{name}" field is None, but this is not allowed'.format(name=name))
leapp.models.fields.ModelViolationError: The value of "target" field is None, but this is not allowed
```

Reply:
@pirat89 We need to fix snactor (n\n-1 patch broke it but we were putting it off) :(

Sorry for this, but testing with snactor won't work at the moment. This will be among my top priorities for the next week though. In the meantime you could use our vagrant box maybe? @MichalHe has explained the details how to do this here https://github.com/oamg/leapp-repository/pull/867#issuecomment-1082313396

Reply:
Created https://github.com/oamg/leapp/issues/771 , all updates will go there.

Reply:
set these envars before running the snactor
```
export LEAPP_UPGRADE_PATH_TARGET_RELEASE=9.0
export LEAPP_UPGRADE_PATH_FLAVOUR=default
```